### ISSUE TITLE
Crash app in android

### ISSUE THREAD

**Issue by ARASHz4** [juliansteenbakker/mobile_scanner#482]
In Mobile Scanner version 3.0.0 app crash when use Mobile Scanner

**Comment by jbrechbuehl** [comment]
Hello,

probably same error here.

Android 13, Flutter 3.3.10, mobile_scanner: 3.0.0 (crash with 3.0.0-beta.4 too) :
```
E/AndroidRuntime(16312): FATAL EXCEPTION: CameraX-core_camera_0
E/AndroidRuntime(16312): Process: com.app_name, PID: 16312
E/AndroidRuntime(16312): java.lang.ExceptionInInitializerError
E/AndroidRuntime(16312):        at androidx.camera.camera2.internal.s.<init>(Unknown Source:149)
E/AndroidRuntime(16312):        at androidx.camera.camera2.internal.O.<init>(Unknown Source:144)
E/AndroidRuntime(16312):        at androidx.camera.camera2.internal.t.b(Unknown Source:34)
E/AndroidRuntime(16312):        at l.G.b(Unknown Source:47)
E/AndroidRuntime(16312):        at androidx.camera.core.F.b(Unknown Source:108)
E/AndroidRuntime(16312):        at androidx.camera.core.E.run(Unknown Source:10)
E/AndroidRuntime(16312):        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
E/AndroidRuntime(16312):        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
E/AndroidRuntime(16312):        at java.lang.Thread.run(Thread.java:1012)
E/AndroidRuntime(16312): Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodException: l.n.values []
E/AndroidRuntime(16312):        at java.lang.Enum.enumValues(Enum.java:272)
E/AndroidRuntime(16312):        at java.lang.Enum.-$$Nest$smenumValues(Unknown Source:0)
E/AndroidRuntime(16312):        at java.lang.Enum$1.create(Enum.java:279)
E/AndroidRuntime(16312):        at java.lang.Enum$1.create(Enum.java:277)
E/AndroidRuntime(16312):        at libcore.util.BasicLruCache.get(BasicLruCache.java:63)
E/AndroidRuntime(16312):        at java.lang.Enum.getSharedConstants(Enum.java:291)
E/AndroidRuntime(16312):        at java.lang.Class.getEnumConstantsShared(Class.java:2461)
E/AndroidRuntime(16312):        at java.util.EnumSet.getUniverse(EnumSet.java:412)
E/AndroidRuntime(16312):        at java.util.EnumSet.noneOf(EnumSet.java:114)
E/AndroidRuntime(16312):        at java.util.EnumSet.of(EnumSet.java:288)
E/AndroidRuntime(16312):        at androidx.camera.camera2.internal.p0.<clinit>(Unknown Source:8)
E/AndroidRuntime(16312):        ... 9 more
E/AndroidRuntime(16312): Caused by: java.lang.NoSuchMethodException: l.n.values []
E/AndroidRuntime(16312):        at java.lang.Class.getMethod(Class.java:2103)
E/AndroidRuntime(16312):        at java.lang.Class.getDeclaredMethod(Class.java:2081)
E/AndroidRuntime(16312):        at java.lang.Enum.enumValues(Enum.java:269)
E/AndroidRuntime(16312):        ... 19 more
```

```dart
Widget _buildQrView(BuildContext context) {
    return RotatedBox(
      quarterTurns: 3,
      child: MobileScanner(
        key: qrKey,
        fit: BoxFit.cover,
        onDetect: (capture) {
          final List<Barcode> barcodes = capture.barcodes;
          for (final barcode in barcodes) {
            log.info('Camera scanned QR code $barcode');
            if (barcode.rawValue != null) {
              bloc.add(Scan(barcode.rawValue!));
            } else {
              log.severe('Camera scanned QR code is null');
            }
          }
        },
      ),
    );
  }
```

No error with version 3.0.0-beta.3

**Comment by GoedertDalmolin** [comment]
Hello, 

I got the same error here:


```
 Widget scannerCamera() {
    return MobileScanner(
        controller: cameraController,
        onDetect: (barcode) {

          setState(() {
            if (barcode.barcodes.isEmpty) {
              if (mounted) {
                setState(() {
                  barcodeLasted = 'Failed to Scan!';
                });
              }
            } else {
              final String code = barcode.barcodes[0].rawValue ?? "";
              debugPrint('Barcode finded! $code');
              if (mounted) {
                setState(() {
                  barcodeLasted = 'Barcode finded! $code';
                });
              }
              if (!isClosed) {
                isClosed = true;
                Navigator.of(context).pop(barcode);
              }
            }
          });
        });
  }
```

**Comment by ntatko** [comment]
In android studio, I'm getting this error:

```
A problem occurred configuring project ':mobile_scanner'.
> Could not resolve all files for configuration ':mobile_scanner:classpath'.
   > Could not resolve com.android.tools.build:gradle:7.4.1.
     Required by:
         project :mobile_scanner
      > No matching variant of com.android.tools.build:gradle:7.4.1 was found. The consumer was configured to find a runtime of a library compatible with Java 8, packaged as a jar, and its dependencies declared externally, as well as attribute 'org.gradle.plugin.api-version' with value '7.5' but:
          - Variant 'apiElements' capability com.android.tools.build:gradle:7.4.1 declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares an API of a component compatible with Java 11 and the consumer needed a runtime of a component compatible with Java 8
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.5')
          - Variant 'javadocElements' capability com.android.tools.build:gradle:7.4.1 declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.5')
          - Variant 'runtimeElements' capability com.android.tools.build:gradle:7.4.1 declares a runtime of a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component compatible with Java 11 and the consumer needed a component compatible with Java 8
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.5')
          - Variant 'sourcesElements' capability com.android.tools.build:gradle:7.4.1 declares a runtime of a component, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java version (required compatibility with Java 8)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.gradle.plugin.api-version (required '7.5')
> Failed to notify project evaluation listener.
   > Could not get unknown property 'android' for project ':mobile_scanner' of type org.gradle.api.Project.
   > Could not get unknown property 'android' for project ':mobile_scanner' of type org.gradle.api.Project.

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
```

Anyone else seeing this?

**Comment by ARASHz4** [comment]
It fixed in 3.1.0
